.github/workflows/tests.yml

```diff
@@ -18,9 +18,13 @@ jobs:
             testbench: ^5.0
           - laravel: ^8.0
             testbench: ^6.0
+          - laravel: ^9.0
+            testbench: ^7.0            
         exclude:
           - php: 7.2
             laravel: ^8.0
+          - php: 7.4
+            laravel: ^9.0            
```

Comment:
PHP 7.3 should also be excluded with Laravel 9.

Reply:
Sorry I've added that now